apps/playground/src/app/modules/components/tile-module/tile-module.component.html

```diff
@@ -25,7 +25,6 @@ <h2>DsfrTileComponent</h2>
     .
   </p>
   <p>It has several <code>&#64;Input()</code> properties:</p>
```

Comment:
OK, for reasons I don't understand, GitHub won't let me put this comment where I want 🤷🏻‍♂️
Anyway, please change line 4 with : `The <code>DsfrTileComponent</code> purpose is to represent a link as a tile
    element.`